public/js/index.js

```diff
@@ -1933,40 +1936,27 @@ $('#snippetExportModalConfirm').click(function () {
 
   if (
     !data.title ||
-    !data.file_name ||
-    !data.code ||
-    !data.visibility_level ||
+    !data.files[0].file_path ||
+    !data.files[0].content ||
     !$('#snippetExportModalProjects').val()
```

Comment:
You execute `$('#snippetExportModalProjects').val()` two times. Here and in L1944.